tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrer.xtend

```diff
@@ -366,7 +369,7 @@ class TclJvmModelInferrer extends AbstractModelInferrer {
 				output.append('''// TODO interaction type '«interaction.name»' does not have a proper method reference''')
 			}
 		} else if (step.componentContext != null) {
-			output.append('''// TODO could not resolve '«step.componentContext.component.name»' - «stepLog»''')
+			output.append('''throw new RuntimeException("Template '«stepLog»' cannot be resolved with any known macro/fixture. Please check your file '«testCase.name».tcl'");''')
```

Comment:
tcl? what about macros or configs?
org.junit.Assert.fail(""); would be possible, too / which one is better and why?
I'd keep the reference to the component, since it is some context information that will make finding this very template easier.
You could even provide line number by using this code:
NodeModelUtils.findActualNodeFor(step).startLine
This would help the user to find the location within the test.